<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="detected_object">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Detected Object</string>
   </property>
  </widget>
  <widget class="QLabel" name="detected_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>60</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>641</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="start_stream">
      <property name="text">
       <string>Start Stream</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stop_stream">
      <property name="text">
       <string>Stop Stream</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="threat_area">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>290</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Threat Area</string>
   </property>
  </widget>
  <widget class="QLabel" name="threat_area_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>330</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
